integration-tests/go.mod

```diff
@@ -7,12 +7,11 @@ require (
 	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b
 	github.com/smartcontractkit/caigo v0.0.0-20230530082629-53a5a4bdb25e
 	github.com/smartcontractkit/chainlink-env v0.3.29
-	github.com/smartcontractkit/chainlink-relay v0.1.7-0.20230531014621-9c303da4c086
+	github.com/smartcontractkit/chainlink-relay v0.1.7-0.20230531175356-03a8f54e3528
 	github.com/smartcontractkit/chainlink-starknet/ops v0.0.0-20230329050701-40e3b18bb026
-	github.com/smartcontractkit/chainlink-starknet/relayer v0.0.0-20230530233948-90c8af98011e
+	github.com/smartcontractkit/chainlink-starknet/relayer v0.0.0-20230531180015-4bfd036299c3
 	github.com/smartcontractkit/chainlink-testing-framework v1.11.5
-	github.com/smartcontractkit/chainlink/integration-tests v0.0.0-20230420131147-ce3c53a39d07
```

Comment:
I'm surprised that this can be dropped, is that right?

Reply:
`go get` removed this for me

```
$ go get github.com/smartcontractkit/chainlink/v2@4c79b5537b9ee63ffb0da3aca72ccaf364bbd7c0
go: warning: github.com/confio/ics23/go@v0.7.0: retracted by module author: subject to the dragonberry vulnerability
go: warning: github.com/cosmos/ibc-go/v4@v4.2.0: retracted by module author: contains huckleberry vulnerability
go: to switch to the latest unretracted version, run:
	go get <module>@latest
go: removed github.com/smartcontractkit/chainlink/integration-tests v0.0.0-20230420131147-ce3c53a39d07
```